src/inet/UDPEndPoint.h

```diff
@@ -249,6 +257,16 @@ class DLL_EXPORT UDPEndPoint : public EndPointBasis
      */
     void Free();
 
+    enum ReleaseDeferralErrorTactic
```

Comment:
nit: A more appropriate word is `strategy`

Reply:
As discussed on slack, we are going to remove `DefferedFree`